data_collection/gazette/spiders/base/portal_da_transparencia.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import re
+
+import dateparser
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PortalDaTransparenciaBaseSpider(BaseGazetteSpider):
+    allowed_domains = ["portaldatransparencia.com.br"]
+    start_date = datetime.date(2000, 1, 1)
```

Comment:
**question**: How did you find this date?

Reply:
It's just a date that is "safe enough" in the context of this platform as there are no gazettes prior to that, but on the other hand it has no performance-related downsides on the way they retrieve the gazettes.